vendor/engines/badges_engine/spec/spec_helper.rb

```diff
@@ -0,0 +1,30 @@
+require 'simplecov'
+ENV["RAILS_ENV"] ||= 'test'
+require File.expand_path("../dummy/config/environment.rb", __FILE__)
+require 'rspec/rails'
+require 'rspec/autorun'
+require 'factory_girl_rails'
+require 'shoulda-matchers'
+require 'haml-rails'
+require 'pry'
+require 'carrierwave'
+
+ENGINE_RAILS_ROOT = File.join(File.dirname(__FILE__), '../')
+Dir[File.join(ENGINE_RAILS_ROOT, 'spec/support/**/*.rb')].each { |f| require f }
+
+ActiveRecord::Migration.check_pending! if defined?(ActiveRecord::Migration)
+
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
+
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+
+  config.use_transactional_fixtures = true
+
+  config.infer_base_class_for_anonymous_controllers = false
+
+  config.order = "random"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.